docs/docs/feature-reference/adding-and-updating-custom-databases.md

```diff
@@ -63,15 +59,18 @@ database, except the `database_name` for the database must already exist.
 
 Arguments:
 
-- `database_path`:  Path or list of paths (local or S3) to file(s) containing the custom database.
+- `database_path`:  Path or list of paths (local or S3) to file(s) containing the custom database. This can be a prefix
+  (i.e., a local directory or an S3 "directory"-style prefix).
 - `tool`: Toolchest tool class with which you use the database (e.g. `toolchest.tools.DiamondBlastp`, 
 `toolchest.tools.Kraken2`).
-- `database_name`: name of the _existing_ database
+- `database_name`: Name of the _existing_ database.
 - `database_primary_name`: If you are uploading multiple database files and your tool takes in a certain file or prefix 
-**instead of the whole directory** as its command-line argument, use this to specify the name of that file or prefix.
+instead of the whole directory as its command-line argument, use this to specify the name of that file or prefix.
+See the "[Specifying `database_primary_name`](adding-and-updating-custom-databases.md#specifying-database_primary_name)" 
+section for more details.
 
-  (For example, if you are uploading files `database.file.1` and `database.file.2`for a tool that takes in 
-`/path/to/database/database.file` as its command-line database, use `database_primary_name=database.file`.)
+  **NOTE**: This argument is **optional** for `update_database`. If omitted, it assumes the same `database_primary_name` as the previous
+version of the custom database.
```

Comment:
Add a mkdocs material [note](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage) or warning instead of "**NOTE**"